ui/src/hooks/useTerminal.ts

```diff
@@ -165,7 +165,6 @@ export const useTerminal = async (el: HTMLElement, option: ICallbackOptions): Pr
         break;
       }
       default: {
-        console.log(JSON.parse(data));
       }
     }
 
```

Comment:
Unfortunately, there's no code provided to analyze. Please provide the code you want to compare.